client/package.json

```diff
@@ -332,7 +336,99 @@
                     "when": "view == blockchainExplorer && viewItem == blockchain-connection-item"
                 }
             ]
-        }
+        },
+        "debuggers": [
+            {
+                "type": "fabric",
```

Comment:
I think we should call this debugger `fabric:node`, `fabric-node`, or something else with `node` in. I think we're going to need separate debuggers for each chaincode language (Go, Java, .NET), each with its own set of configuration properties - trying to combine them all into one might be difficult!